src/test/javascript/spec/component/admin/user-management.component.spec.ts

```diff
@@ -44,31 +71,28 @@ describe('UserManagementComponent', () => {
     // The admin module is lazy loaded - we therefore need a dummy test to load
     // the module and verify that there are no dependency related issues.
     it('should render a component from the admin module', () => {
-        expect(comp).toBeDefined();
+        expect(comp).not.toBe(null);
     });
 
-    it('should parse the user search result into the correct component state', inject(
-        [],
-        fakeAsync(() => {
-            const headers = new HttpHeaders().append('link', 'link;link').append('X-Total-Count', '1');
-            jest.spyOn(service, 'query').mockReturnValue(
-                of(
-                    new HttpResponse({
-                        body: [new User(1)],
-                        headers,
-                    }),
-                ),
-            );
+    it('should parse the user search result into the correct component state', fakeAsync(() => {
+        const headers = new HttpHeaders().append('link', 'link;link').append('X-Total-Count', '1');
+        jest.spyOn(service, 'query').mockReturnValue(
+            of(
+                new HttpResponse({
+                    body: [new User(1)],
+                    headers,
+                }),
+            ),
+        );
 
-            comp.loadAll();
-            // 1 sec of pause, because of the debounce time
-            tick(1000);
+        comp.loadAll();
+        // 1 sec of pause, because of the debounce time
+        tick(1000);
 
-            expect(comp.users && comp.users[0].id).toEqual(1);
-            expect(comp.totalItems).toEqual(1);
-            expect(comp.loadingSearchResult).toEqual(false);
-        }),
-    ));
+        expect(comp.users && comp.users[0].id).toBe(1);
```

Comment:
```suggestion
        expect(comp.users).toHaveLength(1); //or whatever length there should be >0
        expect(comp.users[0].id).toBe(1);
```

Reply:
yes, it's one as there is only one user added. I changed it as suggested.